stdlib/test/builtin/test_hasher.mojo

```diff
@@ -79,8 +85,36 @@ def test_complexe_hash_with_hasher():
     assert_equal(_hash_with_hasher[DummyHasher](hashable), 52)
 
 
+@value
+struct ComplexeHashableStructWithList(_HashableWithHasher):
+    var _value1: SomeHashableStruct
+    var _value2: SomeHashableStruct
+    var _value3: List[UInt8]
```

Comment:
It is quite strange, that other type `List[UInt8]` has the same naming convention `_value*` as of type `SomeHashableStruct`

Reply:
The time we would spend changing or debating this name would be greater than the time lost by readers of the code who will be slightly inconvenienced by it. Furthermore it's not user-facing code, so the impact of a badly chosen name is reduced since less people are going to read this code. I would prefer that we avoid bikeshedding names of dummy variables in unit tests.